pkg/alertmanager/operator.go

```diff
@@ -135,7 +135,7 @@ func New(c prometheusoperator.Config, logger log.Logger, r prometheus.Registerer
 		&monitoringv1.Alertmanager{}, resyncPeriod, cache.Indexers{},
 	)
 	o.metrics.MustRegister(NewAlertmanagerCollector(o.alrtInf.GetStore()))
-	o.metrics.MustRegister(operator.NewStoreCollector("alertmanager", o.alrtInf.GetStore()))
+	o.metrics.MustRegister(operator.NewStoreCollector(monitoringv1.AlertmanagersKind, o.alrtInf.GetStore()))
```

Comment:
Let's start thinking how this can incorporate multiple stores too (not relevant for this PR though) since we plan to move away from multilistwatcher to a native array of informers/stores.

Any ideas? I attempted something in my WIP PR but maybe you have some idea too?